.github/dependabot.yml

```diff
@@ -0,0 +1,4 @@
+version: 2
```

Comment:
Here are the docs https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file